app/data/models/User.py

```diff
@@ -11,6 +13,12 @@ def __init__(self, user_id: int, username: str, email: str, password: str, birth
         self.password = password
         self.birthday = birthday
 
+        user_logger.debug(f"User with id {self.id} has been created")
```

Comment:
add all info, username, email, birthday, but no password(maybe should not log email too) (sensitive info like password and email also token shouldnt be logged I think)